README.md

```diff
@@ -59,76 +53,90 @@ def swagger_spec do
+  swagger_path :show do
+    get "/users/{id}"
+    summary "Get single user"
+    description "Get a single user by id"
+    parameter :id, :path, :integer, "account id", required: true
+    response 200, "Success", :User
+    tag "users"
+  end
+  
+  def show(conn, _params) do
+    user = Repo.get!(MyApp.User, id)
+    render(conn, "show.json", user: user)
+  end
 end
 ```
 
 The `swagger_path` macro takes two parameters:
 
 * The name of the controller action
-* `do` block containing calls into the PhoenixSwagger.Path module.
+* A `do` block containing calls into the `PhoenixSwagger.Path` module.
 
-The body of the DSL is only a thin layer of syntax sugar over a regular phoenix function pipeline.
-The example above can be re-written as:
+The body of the DSL is only a thin layer of syntax sugar over a regular `Phoenix` function pipeline.
+The example above can be rewritten as:
 
 ```elixir
 def swagger_path_index do
   import Phoenix.Swagger.Path
   get("/users")
   |> description("Short description")
-  |> parameter(:query, :id, :integer, "Property id", required: true)
+  |> parameter(:id, :query, :integer, "Property id", required: true)
   |> response(200, "Description")
   |> nest
   |> to_json
 end
 ```
 
-The `do` bock always starts with one of the `get`, `put`, `post`, `delete`, `head`, `options` functions. This creates a new `#SwaggerPath{}` struct to pipeline through the remaining functions.
+The `do` bock always starts with one of the `get`, `put`, `post`, `delete`, `head`, `options` functions. This creates a new `#SwaggerPath{}` struct to pipe it through the remaining functions.
 
-At a minimum, you should probably supply `summary`, `description`, `parameter`, and `response` docs.
+It is recommended that you supply the documentation fields `summary`, `description`, `parameter`, and `response`.
 
-The `parameter` provides description of the routing parameter for the given action and
-may take four positional parameters, and a keyword list of optional parameters:
+The `parameter` option provides the description of a parameter for the given action and may take four positional arguments, and a keyword list of optional arguments:
 
-* The location of the parameter. Possible values are `query`, `header`, `path`, `formData` or `body`. [required];
 * The name of the parameter. [required];
-* The type of the parameter. Allowed only [swagger data types](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types
+* The location of the parameter. Possible values are `query`, `header`, `path`, `formData` or `body`. [required];
+* The type of the parameter. Allowed [swagger data types](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types
 ) [required];
-* Description of a parameter. Can be elixir's `String` or function/0 that returns elixir's string;
+* Description of a parameter. Can be an Elixir `String` or `function/0` that returns an Elixir string;
```

Comment:
I'm not sure if the `function/0` option even works? 
We certainly don't use it that way internally - maybe just remove mention of it?